new project

fauziyahzeeyah opened the pull request.

sajid-munawar reviewed (commented):
# Changes Required 
 ### Hi @fauziyahzeeyah ,
 Nice job, you are almost there.
 
 ![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
 
 ## (Highlights) Good Points: 
 * Followed Github flow :heavy_check_mark:

 ## (Changes Required) Aspects to improve: 
 *  ### Please do the following:
 

#### Merged and Deleted Pull Request ⚠️

* Pull request is merged and deleted. Please do not submit a merged and deleted pull request for review. Kindly revert your merging and deletion or create a new repo for your project.


- [ ] Please kindly add a more descriptive PR title.
 See a demo below:

https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4

- [ ] I kindly suggest that you add a more descriptive PR title and description. 
## EXAMPLE
```
## Hello World
In this Pull Request, I:
- Created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Added a HTML file
- Added a CSS File
- Added a h1 text of "Hello-World"
```


- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

- [ ] Kindly add linters to your project. In the root project, you need to have a folder named .github. Inside this one, you need another folder named workflows. And inside the workflows folder, you need a file named linters.yml. Inside this file, you need to have the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
Here is a video demo I created demostrating  the above:

https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
- [ ] Kindly ensure your linter results pass * AKA: Linter should report NO errors.
- [ ] Kindly create a `.gitignore` file, then add `test.md` and `node_modules` to your .gitgignore file.
> Here is a video demo of the above:

https://user-images.githubusercontent.com/52934128/164221760-0a8ce8da-7bb5-471c-9746-7c4e1cc6c99c.mp4



 ## [Optional] suggestions:
 * Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear
 Please, remember to tag @sajid-munawar in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

fauziyahzeeyah commented:
Hello, how do I send my updated project to you for another review?

On Wed, 26 Oct 2022 at 14:10, Sajid Munawar ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> Changes Required Hi @fauziyahzeeyah <https://github.com/fauziyahzeeyah> ,
>
> Nice job, you are almost there.
>
>
> <https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966>
> (Highlights) Good Points:
>
>    - Followed Github flow ✔️
>
> (Changes Required) Aspects to improve:
>
>    - Please do the following:
>
> Merged and Deleted Pull Request ⚠️
>
>    - Pull request is merged and deleted. Please do not submit a merged
>    and deleted pull request for review. Kindly revert your merging and
>    deletion or create a new repo for your project.
>
>
>    - Please kindly add a more descriptive PR title.
>    See a demo below:
>
>
> https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4
>
>    - I kindly suggest that you add a more descriptive PR title and
>    description.
>
> EXAMPLE
>
> ## Hello World
>
> In this Pull Request, I:
>
> - Created branches with different base branches.
>
> - Followed GitHub flow.
>
> - Added a descriptive and easy to understand README
>
> - Added a HTML file
>
> - Added a CSS File
>
> - Added a h1 text of "Hello-World"
>
>
>
>    -
>
>    Please refer to this article: please read it and apply it in your work
>    from now on: Professional repo - basic rules
>    <https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md>
>    ;
>    -
>
>    Kindly add linters to your project. In the root project, you need to
>    have a folder named .github. Inside this one, you need another folder named
>    workflows. And inside the workflows folder, you need a file named
>    linters.yml. Inside this file, you need to have the linters. The linters
>    needed for this project can be found here
>    <https://github.com/microverseinc/linters-config/tree/master/html-css>;
>    Here is a video demo I created demostrating the above:
>
>
> https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
>
>    - Kindly ensure your linter results pass * AKA: Linter should report
>    NO errors.
>    - Kindly create a .gitignore file, then add test.md and node_modules
>    to your .gitgignore file.
>
> Here is a video demo of the above:
>
>
> https://user-images.githubusercontent.com/52934128/164221760-0a8ce8da-7bb5-471c-9746-7c4e1cc6c99c.mp4
> [Optional] suggestions:
>
>    - Nothing to mention.
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> *Cheers and Happy coding! 👏👏👏*
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear
> Please, remember to tag @sajid-munawar <https://github.com/sajid-munawar>
> in your question so I can receive the notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fauziyahzeeyah/port-folio/pull/1#pullrequestreview-1156476819>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWAVRVR7JDGZQ3EEFKCRMUTWFEUTTANCNFSM6AAAAAARO62CGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sajid-munawar commented:
@fauziyahzeeyah  please go to your dashboard and ask for another review.
